src/demo.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import Suspense from './components/Suspense'
+import { useAsyncResource } from 'use-async-resource'
+
+
+import ReactBarGraph from './react_barchart.jsx'
+// import ReactSupervennStyle from './lib/index.module.css'
+// const ReactSupervenn = React.lazy(() => import('./lib'))
```

Comment:
Your barchart is it would be used in other projects should be implemented in `./lib` (`index.js` is inferred when a js file is not specified).

`demo.jsx` is an "example app" using your component as defined entirely in `lib`